frontend/src/metabase/query_builder/actions.js

```diff
@@ -1554,10 +1554,14 @@ export const loadObjectDetailFKReferences = createThunkAction(
       dispatch.action(CLEAR_OBJECT_DETAIL_FK_REFERENCES);
 
       const state = getState();
+      const tableForeignKeys = getTableForeignKeys(state);
+
+      if (!Array.isArray(tableForeignKeys)) {
+        return null;
+      }
```

Comment:
Just a safety-check, we should be able to leave it when backporting to `master` as the `ObjectDetail` component changes are not going to the 43 release